### PR TITLE
[MASTER] chore(Text): update size and line-height logic to match Figma designs

### PR DESCRIPTION
**Pull request by Kui-kui**
Some candidates gave the feedback that font sizes or line heights didn't really match the values in Figma, so I'm changing them.

### CODE REVIEW

File: src/components/Text/index.tsx [hunk 1]
```diff
@@ -1,13 +1,25 @@
...
 import { TextProps } from "./types";
 
+const getColorVariant = (variant?: string): number => {
+  switch (variant) {
+    case "lighter":
+      return -1;
+    case "light":
+      return 0;
+    case "secondary":
+      return 1;
```
Comment (thanhthaohoang) [inline src/components/Text/index.tsx:12]: Shouldn't be `-1` for secondary and `1` for lighter?

File: src/common/theme.ts [hunk 2]
```diff
@@ -169,21 +169,50 @@ const BASE_COLORS = {
...
+    size(input: number) {
+      switch (input) {
+        case -2:
           return 13;
+        case -1:
+          return 15;
         case 1:
           return 18;
         case 2:
           return 22;
```
Comment (thanhthaohoang) [inline src/common/theme.ts:204]: Shouldn't be `21`?

File: src/components/Text/index.tsx (hunk 1 above)
Comment (Kui-kui) [reply src/components/Text/index.tsx:12]: Thank God you're here 😅

File: src/common/theme.ts (hunk 2 above)
Comment (Kui-kui) [reply src/common/theme.ts:204]: It should, but the interview Figma uses 22...

Comment (thanhthaohoang) [reply src/common/theme.ts:204]: 😱

**Review by thanhthaohoang — APPROVED** [review]
LGTM

**Pull request merged**